pkg/apiserver/slowquery/service.go

```diff
@@ -88,3 +99,66 @@ func (s *Service) getDetails(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, *result)
 }
+
+// @Router /slow_query/download/token [post]
+// @Summary Generate a download token for exported slow query statements
+// @Produce plain
+// @Param request body GetListRequest true "Request body"
+// @Success 200 {string} string "xxx"
+// @Security JwtAuth
+// @Failure 401 {object} utils.APIError "Unauthorized failure"
+func (s *Service) downloadTokenHandler(c *gin.Context) {
+	var req GetListRequest
+	if err := c.ShouldBindJSON(&req); err != nil {
+		utils.MakeInvalidRequestErrorFromError(c, err)
+		return
+	}
+	db := utils.GetTiDBConnection(c)
+	fields := []string{}
+	if strings.TrimSpace(req.Fields) != "" {
+		fields = strings.Split(req.Fields, ",")
+	}
+	list, err := QuerySlowLogList(db, &req)
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+	if len(list) == 0 {
+		utils.MakeInvalidRequestErrorFromError(c, errors.New("no data to export"))
```

Comment:
You can create a new error code for it by using `XxxNs = errorx.NewNamespace(...)` and `ErrXxx = XxxNs.NewType(...)`.

Then, at the FE, you can have this kind of error code translated.

See `pkg/apiserver/configuration/service.go` for an example.

Reply:
Maybe you can also add this feature to statement export if it is missing :)